src/createSurfSplitNavigator.js

```diff
@@ -145,43 +165,46 @@ export const SurfSplitNavigator = ({
         }
         return (
             <NavigationHelpersContext.Provider value={navigation}>
-                <SafeAreaProvider>
+                <SafeAreaProviderCompat>
                     <View style={splitStyles.body}>
                         <View style={splitStyles.main}>
                             {descriptors[mainRoute.key].render()}
                         </View>
                         <View style={splitStyles.detail}>
-                            {state.routes.map((route, index) => {
-                                if (route.key === mainRoute.key) {
-                                    // Do not render the main screen on the detail view
-                                    return null;
-                                }
-
-                                /**
-                                 * Disable the lazy loading, as it breaks
-                                 * the navigation to an unmounted screen!
-                                if (!loaded.includes(index)) {
-                                    // Do not render the screen if we've never navigated to it
-                                    return null;
-                                }
-                                */
-
-                                const descriptor = descriptors[route.key];
-                                const isFocused = state.index === index;
-
-                                return (
-                                    <ResourceSavingScene
-                                        key={route.key}
-                                        style={StyleSheet.absoluteFill}
-                                        isVisible={isFocused}
-                                    >
-                                        {descriptor.render()}
-                                    </ResourceSavingScene>
-                                );
-                            })}
+                            <ScreenContainer style={styles.pages}>
```

Comment:
Can you explain what is it for here?

Reply:
Made it as in bottom tabs - https://github.com/react-navigation/react-navigation/blob/main/packages/bottom-tabs/src/views/BottomTabView.tsx#L43-L63 . Doesn't fix anything, just thought that it might be convenient to make it as in tabs in `react-navigation` and I like the idea behind it with accessibility and theme usage.